release/ray_release/byod/requirements_ml_byod_3.9.in

```diff
@@ -1,31 +1,61 @@
 # Python requirements to run release tests from anyscale byod (gpu type, python 3.9)
 
+-c requirements_compiled.txt
 accelerate
 bitsandbytes
+boto3
+cmake
+crc32c
+cupy-cuda113
 dataset
 datasets
 decord
 deepspeed>=0.12.3
-diffusers
+diffusers==0.12.1
```

Comment:
why is this pinned to 0.12.1 now?

Reply:
for this test to pass https://github.com/ray-project/ray/blob/master/doc/source/serve/tutorials/stable-diffusion.md?plain=1#L9, it doesn't work with the latest version of diffusers

Reply:
could you consult people that know more about the test if this is the right thing to do?

Reply:
CC: @edoakes , @GeneDer is it ok to pin diffusers=0.12.1 for stable diffusion test. For context, the doc tutorial also pins to the same version.

Reply:
Given the doc also pined diffusers to 0.12.1, I think this is the right thing to do. Serve team can follow up to upgrade the api at a later time.